models/stg_shopify__abandoned_checkout.sql

```diff
@@ -45,8 +45,8 @@ final as (
         billing_address_zip,
         buyer_accepts_marketing as has_buyer_accepted_marketing,
         cart_token,
-        cast(closed_at as {{ dbt.type_timestamp() }}) as closed_at,
-        cast(created_at as {{ dbt.type_timestamp() }}) as created_at,
+        {{ dbt_date.convert_timezone(column='cast(created_at as ' ~ dbt.type_timestamp() ~ ')', target_tz=var('shopify_timezone', "UTC"), source_tz="UTC") }} as created_at,
```

Comment:
Not a big issue, but it seems as if `source_tz` is set to UTC by default, so wonder if `source_tz` is needed? https://github.com/calogica/dbt-date/blob/main/macros/calendar_date/convert_timezone.sql

Reply:
Doesn't need to be addressed here though.